src/components/SimpleButton.js

```diff
@@ -28,8 +29,8 @@ export default function SimpleButton(props) {
   const [style, setStyle] = useState({});
 
   useEffect(() => {
-    updateStyle('white', baseColor, additionalStyle, false, disabled);
-  }, [baseColor, additionalStyle, disabled]);
+    updateStyle(baseColor, fontColor, additionalStyle, false, disabled);
+  }, [baseColor, additionalStyle, disabled, fontColor]);
```

Comment:
Codacy has a fix for the issue: [React Hook useEffect has a missing dependency: 'updateStyle'. Either include it or remove the dependency array.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9165245)
```suggestion
  }, [baseColor, additionalStyle, disabled, fontColor, updateStyle]);
```